cfme/configure/configuration/region_settings.py

```diff
@@ -870,3 +872,172 @@ class CANDUCollectionDetails(CFMENavigateStep):
 
     def step(self):
         self.prerequisite_view.candu_collection.select()
+
+
+# ========================= Replication ================================
+
+class ReplicationView(RegionView):
+    """ Replication Tab View """
+    replication_type = BootstrapSelect(id='replication_type')
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return self.replication_type.is_displayed
```

Comment:
**Question**

What about an additional check for the correct Region?

Reply:
Good point, thank you. I've missed that. Will add a check for the Region